client/post-editor/editor-ground-control/index.jsx

```diff
@@ -67,28 +66,28 @@ export default React.createClass( {
 	},
 
 	componentDidMount: function() {
-		if ( ! config( 'email_verification_gate' ) ) return;
+		if ( !this.props.user ) {
```

Comment:
Though I am aware of it will be gone in no time, I think it would look better if we add a space following `!` before we tackled #6256.